ubuntu-1804-image_5b461f730428630abc0bf442.md

```diff
@@ -1,4 +1,13 @@
-- [Overview](#overview)
+The `ubuntu1804` is a customized image based on [Ubuntu 18.04 LTS](https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes)
+optimized for CI/CD. It comes with a set of preinstalled languages, databases,
+and utility tools commonly used for CI/CD workflows. The image can be paired
+with any [Linux machine type][machine-types] when defining the [agent][agent]
+of your pipeline or block.
+
+The `ubuntu1804` is a virtual machine image. The user in the environment, named
```

Comment:
I propose saying "virtual machine (VM)" here, and than using "VM" instead of "virtual machine" in further text.